ci/slt

```diff
@@ -0,0 +1 @@
+sql-logic-tests
```

Comment:
We call this `sqllogictest` elsewhere (test/sqllogictest), and indeed the correct name of this program is `sqllogictest` (no hyphens). On this one, can we adjust Buildkite to use the same spelling?